packages/dashmate/templates/platform/dapi/envoy/envoy.yaml.dot

```diff
@@ -7,257 +7,282 @@ static_resources:
-                          route:
-                            cluster: core_and_platform
-                        # health-check endpoints
-                        - match:
-                            prefix: "/grpc.health.v1.Health"
-                          route:
-                            cluster: tx_filter_stream
-                        # configuration of the static responses of unsupported api versions
-                        # core static response
-                        - match:
-                            safe_regex:
-                              google_re2: {}
-                              regex: "\/org\\.dash\\.platform\\.dapi\\.v[1-9]+\\.Core"
-                          response_headers_to_add:
-                            - header:
-                                key: "Content-Type"
-                                value: "application/grpc-web+proto"
-                            - header:
-                                key: "grpc-status"
-                                value: "12"
-                            - header:
-                                key: "grpc-message"
-                                value: "Specified service version is not supported"
-                          direct_response:
-                            status: 204
-                        # platform static response
-                        - match:
-                            safe_regex:
-                              google_re2: {}
-                              regex: "\/org\\.dash\\.platform\\.dapi\\.v[1-9]+\\.Platform"
-                          response_headers_to_add:
-                            - header:
-                                key: "Content-Type"
-                                value: "application/grpc-web+proto"
-                            - header:
-                                key: "grpc-status"
-                                value: "12"
-                            - header:
-                                key: "grpc-message"
-                                value: "Specified service version is not supported"
-                          direct_response:
-                            status: 204
-                        # JSON RPC endpoints
-                        - match:
-                            path: "/"
-                          route:
-                            cluster: json_rpc
-                      cors:
-                        allow_origin_string_match:
-                          - prefix: "*"
-                        allow_methods: GET, PUT, DELETE, POST, OPTIONS
-                        allow_headers: keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,custom-header-1,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout
-                        max_age: "1728000"
-                        expose_headers: custom-header-1,grpc-status,grpc-message
+          - name: envoy.http_connection_manager
```

Comment:
Consider to remove duplicates https://www.envoyproxy.io/docs/envoy/latest/configuration/overview/examples#special-yaml-usage